AK/Random.h

```diff
@@ -47,6 +47,17 @@ inline void fill_with_random(void* buffer, size_t length)
 {
 #if defined(__serenity__)
     arc4random_buf(buffer, length);
+#elif defined (OSS_FUZZ)
+    // Ugly hack to get OSS-Fuzz environment working, where we dont have the
+    // getentropy call. We use the conditional because we need buffer and
+    // length to be used.
+    int rc;
+    if (buffer && length) { 
+        rc = -1;
+    } 
+    else {
+        rc = -1;
```

Comment:
Wouldn't just
```suggestion
    (void)buffer;
    (void)length;
```
do the trick?

Reply:
It did - awesome, thanks!